input/resources/au-practitioner.xml

```diff
@@ -89,8 +89,8 @@
     <element id="Practitioner.extension:recordedSexOrGender">
       <path value="Practitioner.extension"/>
       <sliceName value="recordedSexOrGender"/>
-      <short value="Recorded sex or gender type shall be a member of the AU Recorded Sex or Gender Type value set if any codes within that value set can apply. Recorded sex or gender source document type shall be a member of the AU Recorded Sex or Gender (RSG) Source Document Type value set if any codes within that value set can apply. Recorded sex or gender jurisdiction shall be a member of the AU Recorded Sex or Gender (RSG) Source Document Jurisdiction value set if any codes within that value set can apply"/>
-      <comment value="Person Recorded Sex Or Gender type [AU Recorded Sex or Gender Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender source document [AU Recorded Sex or Gender (RSG) Source Document Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender jurisdiction [AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-jurisdiction.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))."/>
+      <short value="Recorded sex or gender type shall be a member of the AU Recorded Sex or Gender Type value set if any codes within that value set can apply. Recorded sex or gender source document type shall be a member of the AU Recorded Sex or Gender (RSG) Source Document Type value set if any codes within that value set can apply. Recorded sex or gender jurisdiction shall be a member of the Jurisdiction - AU Extended value set if any codes within that value set can apply"/>
+      <comment value="Person Recorded Sex Or Gender type [AU Recorded Sex or Gender Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender source document [AU Recorded Sex or Gender (RSG) Source Document Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender jurisdiction [Jurisdiction - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/valueset-au-jurisdiction-extended.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))."/>
```

Comment:
IG publisher isn't recognising the url with that case. Replace with "ValueSet-au-jurisdiction-extended.html"

Reply:
Updated valueset page name to Title case